app/javascript/packs/components/Trail.vue

```diff
@@ -1,40 +1,59 @@
 <template>
-    <v-card>
-      <v-card-media 
-        class="white--text" 
-        height="200px" 
-        src="https://cdn.vuetifyjs.com/images/cards/docks.jpg"
-      >
-        <v-container fill-height fluid>
-          <v-layout fill-height>
-            <v-flex xs12 align-end flexbox>
-              <span class="headline">{{ title }}</span>
-            </v-flex>
-          </v-layout>
-        </v-container>
-      </v-card-media>
-      <v-card-actions>
-        <v-btn icon>
-          <v-icon>favorite</v-icon>
+  <v-card>
+    <v-card-media
+      class="white--text"
+      height="200px"
+      src="https://cdn.vuetifyjs.com/images/cards/docks.jpg"
```

Comment:
WDYT?

We can pass the `src` by props and keep by default `"https://cdn.vuetifyjs.com/images/cards/docks.jpg"`